buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -221,8 +252,8 @@ class ClusterFormationTasks {
         Map esConfig = [
                 'cluster.name'                 : node.clusterName,
                 'pidfile'                      : node.pidFile,
-                'path.repo'                    : "${node.homeDir}/repo",
-                'path.shared_data'             : "${node.homeDir}/../",
+                'path.repo'                    : "${node.baseDir}/../repo",
+                'path.shared_data'             : "${node.baseDir}/../",
```

Comment:
This seems like a dangerous value since it would mean each node would have read/write access to every other node? Should we use build/cluster/shared for this, separate from the build/cluster/nodeN we have for each node?